Pipfile

```diff
@@ -8,6 +8,8 @@ django = "*"
 
 [dev-packages]
 flake8 = "*"
+pytest = "*"
```

Comment:
We don't need to add pytest, its sub dependence of pytest-django

Reply:
Even if its a sub dependency, when used explicitly its best practice to put the dependency you are using in the pipfile/package.json/maven etc